UI/window-basic-status-bar.hpp

```diff
@@ -110,4 +112,6 @@ private slots:
 	void RecordingUnpaused();
 
 	void ReconnectClear();
+
+	void ShowMessage(const QString &message, int timeout = 0);
```

Comment:
Why not just keep the original names to avoid having to change every instance of it?